landscape/client/configuration.py

```diff
@@ -805,7 +819,7 @@ def report_registration_outcome(what_happened, print=print):
     human-readable form.
     """
     messages = {
-        "success": "System successfully registered.",
+        "success": "Registration request sent successfully.",
```

Comment:
Oh thank you for adjusting this to be more accurate.

Reply:
You can thank Rajan